testgrid/config.yaml

```diff
@@ -11,6 +11,7 @@ dashboards:
 - name: googleoss-kubeflow-gcp-blueprints
 - name: googleoss-grpc-transcoder
 - name: googleoss-http-pattern-matcher
+- name: googleoss-kpt-config-sync
 
```

Comment:
Does the dashboard group include all the prow jobs? I'm wondering how is it configured.

Reply:
To be honest I do not know all of the details about how this is configured. I put this in here to satisfy a presubmit check which was failing because there was a periodic job without an assigned dashboard.
```
* configuration error for (TestGroup) prober-runner-sa-key-rotate: Each Test Group must be referenced by at least 1 Dashboard Tab.
```

The presubmit jobs did not need to be annotated with a dashboard, so I left them unchanged. After doing some searching around it appears that the testgrid can be found [here](https://testgrid.k8s.io/googleoss). I think if we want the presubmit jobs to also show up on this dashboard that we can add them, but currently they do not have the dashboard annotation.

Reply:
It is okay to not include the presubmit jobs in the dashboard as the result should show up in each PR. 
Just for my curiosity, are we configuring the postsubmit jobs in the same testgrid?

Reply:
We weren't planning to include any postsubmit jobs as of now, but if we did add postsubmit jobs I expect we would configure them in the same testgrid.

Reply:
ACK.